src/ripple/protocol/STValidation.h

```diff
@@ -171,8 +171,7 @@ STValidation::STValidation(
 {
     if (checkSignature && !isValid())
     {
-        JLOG(debugLog().error()) << "Invalid signature in validation: "
-                                 << getJson(JsonOptions::none);
+        JLOG(debugLog().error()) << "Invalid signature in validation";
```

Comment:
I still want the hex if not the json dumped under this condition, otherwise we wouldnt know which validation/signature was invalid if we had to debug with this

Reply:
I agree I marked it as DNM. Still figuring out the solution.